DigitalLearningSolutions.Data/Services/UserService.cs

```diff
@@ -182,46 +190,50 @@ List<DelegateUser> delegateUsers
         }
 
         // TODO HEEDLS-887 Make sure this logic is correct with the new account structure
-        public void UpdateUserAccountDetailsForAllVerifiedUsers(
+        public void UpdateUserAccountDetailsForAllUsers(

```

Comment:
This method name sounds to me like it's going to update all Users in the database. Maybe something like `UpdateUserDetailsForAllLinkedAccounts` would be clearer?

Reply:
I'm not sure about that name either, as we have no concept of linked accounts like we used to. I'd probably prefer something like `UpdateUserDetailsAndCentreSpecificDetails`